influxql/functions.go

```diff
@@ -41,16 +42,25 @@ func InitializeMapFunc(c *Call) (MapFunc, error) {
 	// Ensure that there is either a single argument or if for percentile, two
 	if c.Name == "percentile" {
 		if len(c.Args) != 2 {
-			return nil, fmt.Errorf("expected two arguments for percentile()")
+			return nil, fmt.Errorf("expected two arguments for %s()", c.Name)
+		}
+	} else if strings.HasSuffix(c.Name, "derivative") {
```

Comment:
Oh yeah, since this is used so much, definitely worth adding `IsDerivative()` to functions.go. Next time.